src/latex/fls-macros.sty

```diff
@@ -1,108 +1,467 @@
 \NeedsTeXFormat{LaTeX2e}
-\ProvidesPackage{fls-macros}[2023/08/23]
+\ProvidesPackage{agda-macros}[2023/08/22]
```

Comment:
If you rename this, I think you should rename the file as well. I think the name is better, but maybe `agda-latex-macros.sty` would be even better?